h5p-bildetema-words-theme-image/src/components/ThemeImageContainer/ThemeImageContainer.module.scss

```
@@ -1,7 +1,7 @@
 @use "../../../../common/abstracts" as *;
 
 .wrapper {
-  display: flex;
+  display: -webkit-box;
   flex-direction: column;
```

This has the same effect and works in all browsers.

```suggestion
  display: flex;
```

---

ok